### ISSUE TITLE
support `-t` and `-w` options before required arguments?

### ISSUE THREAD

**Issue by lsloan** [prasmussen/chrome-cli#76]
To follow Unix/Linux CLI style a little more closely, could you add support for the options `-t`, `-w`, etc. to appear **_before_** the required positional arguments?  That is, currently the `open` command only allows the `-w` option **_after_** the URL positional argument, like this…

```sh
chrome-cli open url -w id
```

It would be helpful, especially to users with years of Unix/Linux CLI experience, if the `-w` option would be allowed **_before_** the URL positional argument, like this…

```sh
chrome-cli open -w id url
```

And the same would apply to all other CLI options for all of the commands.

It could be implemented to allow the options before **_OR_** after the positional arguments, making it convenient and backwards compatible with the current syntax.

**Comment by prasmussen** [comment]
I totally agree with this. If I ever rewrite this tool in a more modern language I will use a proper args lib that can handle this.
